roles/chameleon_usage/tasks/main.yml

```diff
@@ -69,7 +69,7 @@
 
 - name: Fill in up-to-date data to chameleon_usage
   become: yes
-  command: "/usr/local/sbin/chameleon-usage-extract-chi-data"
+  command: "/usr/local/sbin/chameleon-usage-extract-chi-data &"
```

Comment:
As this depends on the mysqlbackup information, should it be run inside a container with that volume mounted?